app/scrapers/__init__.py

```diff
@@ -4,11 +4,14 @@
 from duckduckgo import Duckduckgo
 from bing import Bing
 from yahoo import Yahoo
+from ask import Ask
+
 scrapers = {
         'g':Google(),
         'b':Bing(),
         'y':Yahoo(),
         'd':Duckduckgo(),
+	'a':Ask(),
```

Comment:
Remove ',' (comma) after `'a':Ask()`
Although it does not cause any error but it would still be better to remove it.